examples/translation-job.json

```diff
@@ -0,0 +1,17 @@
+{
+    "id": "cf5e1c7c-86ae-4f03-aa9e-513b46cd3b7e",
+    "created_on": "2016-10-11T12:11:32+02:00",
```

Comment:
We should really standardise on a single format for the timezone. Is it "+0200" or "+02:00"?